### PR TITLE
MOTECH-2833: Format manipulation modal buttons are not responding

### PR DESCRIPTION
**Pull request by palbecki**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by kmadej** [comment]
test this please

File: modules/tasks/tasks/src/main/resources/webapp/js/directives.js [hunk 1]
```diff
@@ -767,12 +762,23 @@
...
 
-    directives.directive('formatManipulationButton', ['$compile', 'ModalFactory', 'BootstrapDialogManager', 'LoadingModal', function ($compile, ModalFactory, BootstrapDialogManager, LoadingModal) {
+    directives.directive('formatManipulationButton', ['$compile', function ($compile) {
         return {
             restrict: 'EA',
             templateUrl: '../tasks/partials/widgets/string-manipulation-format-button.html',
             link: function (scope, element, attrs) {
-                var modalScope = scope.$new();
+                var modalScope = scope.$new(),
+                    _parse = function (value) {
```
Comment (mkwiatkowskisoldevelo) [inline modules/tasks/tasks/src/main/resources/webapp/js/directives.js]: please change _parse to parseFormatInput or something other without '_'

File: modules/tasks/tasks/src/main/resources/webapp/js/directives.js [hunk 2]
```diff
@@ -809,48 +812,23 @@
...
-                    value.forEach(function(obj){
-                        if(obj.value){
-                            arr.push(obj.value);
-                        }
-                    });
-                    return arr.join(",");
-                });
-                ngModel.$formatters.push(function (value) {
+            link: function (scope, el, attrs) {
+                var _format = function (value) {
```
Comment (mkwiatkowskisoldevelo) [inline modules/tasks/tasks/src/main/resources/webapp/js/directives.js]: please change `_format` to `formatContent` or something other without '_'

**Comment by mkwiatkowskisoldevelo** [comment]
test this please

**Comment by mkwiatkowskisoldevelo** [comment]
@kmadej looks good to me

**Comment by kmadej** [comment]
test this please

**Pull request merged**